scss/atoms/_tables.scss

```diff
@@ -0,0 +1,69 @@
+// ==========================================================================
+// Typography
+// ==========================================================================
+
+table {
+	width: 100%;
+	margin-bottom: em( 20px, 16px );
+	font-size: em( 16px );
+	text-align: left;
+}
+
+thead {
+	border-bottom: 1px solid #eee;
+
+	> tr > th {
+		border-top: 0;
+	}
+}
+
+td,
+th {
+	border-top: 1px solid #eee;
+	padding: em( 12px, 16px );
+	white-space: nowrap;
+}
+
+th {
+	padding: em( 12px, 12px );
+	color: #666;
+	font-weight: 400;
+	font-size: em( 12px, 16px );
+	text-transform: uppercase;
```

Comment:
At least my self ( not sure about @cvrebert ) I was just talking about forcing `uppercase`